org.ioquake3.IOQuake3.yaml

```diff
@@ -1,35 +1,47 @@
 ---
```

Comment:
As I mentioned, Flatpak yaml manifests do not start with `---` and although it will most likely work fine, it is still not correct.